### PR TITLE
Added routes

### PR DESCRIPTION
**Pull request by Akeron44**
I implemented the Private Routes, the NavBar and also getTokenFromLocalStorage, isAdminAuthenticated and isUserAuthenticated functions.

### CODE REVIEW

File: src/utils/auth.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { jwtDecode, JwtPayload } from "jwt-decode";
+
+interface ExtendedJwtPayload extends JwtPayload {
+  email: string;
+}
+
+export const getTokenFromLocalStorage = () => {
+  // const token = localStorage.getItem("token") || '';
+  const token ="eyJhbGciOiJSUzI1NiIsImtpZCI6IjBjYjQyNzQyYWU1OGY0ZGE0NjdiY2RhZWE0Yjk1YTI5ZmJhMGM1ZjkiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJodHRwczovL3NlY3VyZXRva2VuLmdvb2dsZS5jb20vbG9naW4tYzE2ZTMiLCJhdWQiOiJsb2dpbi1jMTZlMyIsImF1dGhfdGltZSI6MTcyMTY4Njg4OSwidXNlcl9pZCI6IlVwMWpYcURzdDVmQ3Zzc1JpSVJ1WWtDVDgxRjIiLCJzdWIiOiJVcDFqWHFEc3Q1ZkN2c3NSaUlSdVlrQ1Q4MUYyIiwiaWF0IjoxNzIxNjg2ODg5LCJleHAiOjE3MjE2OTA0ODksImVtYWlsIjoiYmFuaUBnbWFpbC5jb20iLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnsiZW1haWwiOlsiYmFuaUBnbWFpbC5jb20iXX0sInNpZ25faW5fcHJvdmlkZXIiOiJwYXNzd29yZCJ9fQ.O-2yfEH0Lcgwz5Uw1e4x1tiumgqkbbUuWcTXbNvE7dy_KwZYkMlVYzlBpvzJh0l6g3Rd89ATsxemJBOVUO_ku-AYt1SnsJ03sYEAaYnz1S87wqr6CTCv35X9fn1gbhLYbvkjjHVeg50UAVwQPQMj4W7wZbo13uRC_0D3o5slYfaYWF9LboKaOuX-tYMI2RzBglwoskFcCmuZ3CezP34Je3ETrYMn8gaoAuio0hFaEBh-i3Hf2NQgQWKX258JAkHA-js9IKZH2iLb-jJLomIy1K9lSOK_YoJ-OBvnUqTDiihSw9Bi7FSLslCPZ_xzqkDB2_DLS5BjcM0h84z8NhqYdA"
```
Comment (gerimujo496) [inline src/utils/auth.ts]: remove this

File: src/components/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { HomeOutlined, ShoppingCartOutlined, LogoutOutlined, UserOutlined } from "@ant-design/icons";
+import type { MenuProps } from "antd";
+import { Menu } from "antd";
+import { Link } from "react-router-dom";
+import { isAdminAuthenticated } from "../utils/auth";
+
+type MenuItem = Required<MenuProps>["items"][number];
+
+const items: MenuItem[] = [
```
Comment (gerimujo496) [inline src/components/NavBar.tsx:10]: duhet te jet me import export kjo dhe ne vend te key duhete te jet id  dhe kur te peroresh duet te jeg funksion qe e ndryshon nga key ne id<ma ka mesuar kosta
>

File: src/utils/auth.ts (hunk 1 above)
Comment (danielaKokoneshi309) [inline src/utils/auth.ts]: remove this

File: src/index.css [hunk 3]
```diff
@@ -13,6 +13,7 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
+
```
Comment (danielaKokoneshi309) [inline src/index.css]: ?

Comment (Akeron44) [reply src/index.css]: Removed it

File: src/utils/auth.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import { jwtDecode, JwtPayload } from "jwt-decode";
+
+interface ExtendedJwtPayload extends JwtPayload {
+  email: string;
+}
+
+export const getTokenFromLocalStorage = () => {
+  // const token = localStorage.getItem("token") || '';
```
Comment (Akeron44) [inline src/utils/auth.ts]: Remove OR and check if the token does not exist. If it does not exist return null

**Review by gerimujo496 — APPROVED** [review]

**Review by danielaKokoneshi309 — APPROVED** [review]

**Pull request merged**